plone/app/widgets/tests/robot/test_select_widget.robot

```diff
@@ -115,7 +115,7 @@ Text matches Input
 
 Check Class On Element
   [Arguments]  ${locator}
-  ${class_value} =  Get Element Attribute  identifier=${locator}@class
+  ${class_value} =  Get Element Attribute  identifier=${locator}  class
```

Comment:
This change seems to fail in Plone 5.1  with `Keyword 'Selenium2Library.Get Element Attribute' expected 1 argument, got 2.` (https://jenkins.plone.org/job/pull-request-5.1/1477/testReport)

Is this branch only for 5.2 or is this a error?